web/src/hotkey.js

```diff
@@ -129,56 +131,56 @@ const keydown_either_mappings = {
+    61: { name: "upvote_first_emoji", message_view_only: true }, // '='
+    45: { name: "toggle_message_collapse", message_view_only: true }, // '-'
+    47: { name: "search", message_view_only: false }, // '/'
+    58: { name: "toggle_reactions_popover", message_view_only: true }, // ':'
+    62: { name: "compose_quote_reply", message_view_only: true }, // '>'
+    63: { name: "show_shortcuts", message_view_only: false }, // '?'
+    64: { name: "compose_reply_with_mention", message_view_only: true }, // '@'
+    65: { name: "stream_cycle_backward", message_view_only: true }, // 'A'
+    67: { name: "C_deprecated", message_view_only: true }, // 'C'
+    68: { name: "stream_cycle_forward", message_view_only: true }, // 'D'
+    71: { name: "G_end", message_view_only: true }, // 'G'
+    73: { name: "open_inbox", message_view_only: true }, // 'I'
+    74: { name: "vim_page_down", message_view_only: true }, // 'J'
+    75: { name: "vim_page_up", message_view_only: true }, // 'K'
+    77: { name: "toggle_topic_visibility_policy", message_view_only: true }, // 'M'
+    80: { name: "narrow_private", message_view_only: true }, // 'P'
+    82: { name: "respond_to_author", message_view_only: true }, // 'R'
+    83: { name: "toggle_stream_subscription", message_view_only: true }, // 'S'
+    85: { name: "mark_unread", message_view_only: true }, // 'U'
+    86: { name: "view_selected_stream", message_view_only: false }, // 'V'
+    97: { name: "all_messages", message_view_only: true }, // 'a'
+    99: { name: "compose", message_view_only: true }, // 'c'
+    100: { name: "open_drafts", message_view_only: true }, // 'd'
+    101: { name: "edit_message", message_view_only: true }, // 'e'
+    103: { name: "gear_menu", message_view_only: true }, // 'g'
+    104: { name: "vim_left", message_view_only: true }, // 'h'
+    105: { name: "message_actions", message_view_only: true }, // 'i'
+    106: { name: "vim_down", message_view_only: true }, // 'j'
+    107: { name: "vim_up", message_view_only: true }, // 'k'
+    108: { name: "vim_right", message_view_only: true }, // 'l'
+    109: { name: "move_message", message_view_only: true }, // 'm'
+    110: { name: "n_key", message_view_only: false }, // 'n'
+    112: { name: "p_key", message_view_only: false }, // 'p'
+    113: { name: "query_streams", message_view_only: true }, // 'q'
+    114: { name: "reply_message", message_view_only: true }, // 'r'
+    115: { name: "toggle_conversation_view", message_view_only: true }, // 's'
+    116: { name: "open_recent_view", message_view_only: true }, // 't'
+    117: { name: "toggle_sender_info", message_view_only: true }, // 'u'
+    118: { name: "show_lightbox", message_view_only: true }, // 'v'
+    119: { name: "query_users", message_view_only: true }, // 'w'
+    120: { name: "compose_private_message", message_view_only: true }, // 'x'
+    122: { name: "zoom_to_message_near", message_view_only: true }, // 'z'
```

Comment:
Please don't autoformat existing open source code to your preferred style -- you probably need to turn off an editor configuration feature that did this.

Reply:
understood. Will get to it immediately @timabbott